A32NX/html_ui/Pages/A32NX_Core/A32NX_LocalVarUpdater.js

```diff
@@ -189,5 +194,20 @@ class A32NX_LocalVarUpdater {
         return this.isPacksOneSupplying;
     }
 
+    _areSlidesArmed() {
+        const cabin = SimVar.GetSimVarValue('INTERACTIVE POINT OPEN:0', 'percent');
+        const catering = SimVar.GetSimVarValue('INTERACTIVE POINT OPEN:3', 'percent');
+        const cargo = SimVar.GetSimVarValue('INTERACTIVE POINT OPEN:5', 'percent');
+        const beacon = SimVar.GetSimVarValue('LIGHT BEACON ON', 'bool');
+        const onground = SimVar.GetSimVarValue('SIM ON GROUND', 'bool');
+        const onrw = SimVar.GetSimVarValue('ON ANY RUNWAY', 'bool');
+
+        if ((cabin < 5 && catering < 5 && cargo < 5 && beacon) || !onground || onrw) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
To reduce the number of SimVar calls you could rewrite this in a way where it returns early as soon as any of these is true (or false respectively). For example, when SIM ON GROUND is false there is reason to check any other SimVar at all, thus saving 5 uncessary SimVar calls. The code is more verbose and not so nice, but it's a performance improvement.

Reply:
Yeah, kind of, the new approach works.